framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/hash_join_date.q

```diff
@@ -0,0 +1,16 @@
+alter session set `exec.hashjoin.enable.runtime_filter` = true;
+alter session set `exec.hashjoin.runtime_filter.max.waiting.time` = 10000;
+
+select
+  count(*) as row_count
+from
+  lineitem l1
+where
+  l1.l_shipdate IN (
```

Comment:
The list of IN elements will be greater than 20 (about 2.5K values), so this should switch to an implicit join. Once stats get enabled, expect that the join will become a broadcast join.

Reply:
Okay. Will disable broadcast join for this case.